pkgs/applications/editors/vscode/generic.nix

```
@@ -71,6 +71,8 @@ in
       if system == "x86_64-darwin" then ''
         mkdir -p "$out/Applications/${longName}.app" $out/bin
         cp -r ./* "$out/Applications/${longName}.app"
+
+        # vscodium's macOS release zip dosn't rename the code exectuable properly.
```

this applies to vscode also right?

---

vscode also has the executable named `code` yes.

---

I was just asking because it says `vscodium`
```suggestion
        # macOS release zip dosn't rename the code exectuable properly.
```

---

The rename only applies to vscodium, and only for the macOS release zip. The vscode package doesn't need the rename, nor does the linux package of vscodium, so I felt it better to err on the side of verbosity.